test/test_jit.py

```diff
@@ -12029,6 +12029,15 @@ def fn(x):
 
         self.checkScript(fn, ("abcde",))
 
+    def test_str_cmp(self):
+        def test(a, b):
+            # type: (str, str) -> Tuple[bool, bool, bool, bool, bool, bool]
+            return a != b, a == b, a < b, a > b, a <= b, a >= b
+
+        self.checkScript(test, ("1", "2"))
```

Comment:
will this match python's string comparison semantics? Can we try some more complicated comparisons just to make sure it's consistent

Reply:
lexicographic ordering is the same between c++ & python